app/views/devise/unlocks/new.html.erb

```diff
@@ -7,22 +7,22 @@
         <h1 class="h4 text-gray-900 mb-4"><%= t(".resend_unlock_instructions") %></h1>
       </div>
 
-      <%= turbo_frame_tag resource do %>
-    <%= simple_form_for(resource, as: resource_name, url: unlock_path(resource_name), html: { method: :post }) do |f| %>
-        <%= f.error_notification %>
-        <%= f.full_error :unlock_token %>
+      <%= turbo_frame_tag :new_user, data: { controller: "turbo-form-submit-redirect" } do %>
```

Comment:
In my opinion, you can use a partial with this form. Idk if you want to add an edit method, but if you do it, it will simplify the code. If not, dont listen me, im trying to be smart